lib/stimulus_reflex/broadcaster.rb

```diff
@@ -0,0 +1,57 @@
+module StimulusReflex
+  module Broadcaster
+    def self.included klass
```

Comment:
I prefer parens on method definition arguments as a general rule. 

Also, I'd prefer to make this a bit more idiomatic Rails-like code by including `ActiveSupport::Concern` and then using `module ClassMethods` to define the class methods.

Reply:
Fortunately or unfortunately, I am 100% reliant on @julianrubisch for this ask. The words are familiar but I don’t know how to do that.

Reply:
Could you write that out as a suggestion? I’m not completely sure I understand, either